econml/cate_estimator.py

```diff
@@ -9,7 +9,8 @@
 from copy import deepcopy
 from warnings import warn
 from .inference import BootstrapInference
-from .utilities import tensordot, ndim, reshape, shape, parse_final_model_params, inverse_onehot, Summary
+from .utilities import tensordot, ndim, reshape, shape, parse_final_model_params,\
+    inverse_onehot, Summary, get_input_columns
```

Comment:
Rather than escaping the line break, I believe that this style is preferred:

```suggestion
from .utilities import (tensordot, ndim, reshape, shape, parse_final_model_params,
                        inverse_onehot, Summary, get_input_columns)
```